includes/class-wc-customer-download-log.php

```diff
@@ -0,0 +1,242 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class for customer download logs.
+ *
+ * @version		3.3.0
+ * @since		3.3.0
+ * @package		WooCommerce/Classes
+ * @author		WooThemes
+ */
+class WC_Customer_Download_Log extends WC_Data implements ArrayAccess {
```

Comment:
@mikejolley good question.  It seemed that all `WC_Data` classes had this interface implemented (e.g. `WC_Customer_Download` which I was basing this on), so I assumed there was a spot in the codebase that required this.  I'll try removing this and will make sure tests and such still pass.